setup/mod/rtkernel.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
"code" type files, like scripts, should include the standard pi-stomp header (see setup/audio/audiocard-setup.sh)

Why is this in setup/mod?  RTK is independent of MOD.  Makes more sense to be in setup/sys.

Similar with the linux deb file.  If that file is the same as the one in michavdm 64-bit-rtk, when I run rtkernel.sh, I get the error:

+ sudo apt install ./linux-image-5.15.12-rt25-v8+_5.15.12-1_arm64.deb
Reading package lists... Done
E: Unsupported file ./linux-image-5.15.12-rt25-v8+_5.15.12-1_arm64.deb given on commandline

Reply:
I have moved the script to sys folder and re uploaded the deb file. This is the deb file I used a few days ago, so should be good now, unless GitHub upload is breaking it.